lib/grim/image_magick_processor.rb

```diff
@@ -21,14 +21,30 @@ def count(path)
     end
 
     def save(pdf, index, path, options)
-      width   = options.fetch(:width,   Grim::WIDTH)
-      density = options.fetch(:density, Grim::DENSITY)
-      quality = options.fetch(:quality, Grim::QUALITY)
+      width      = options.fetch(:width,   Grim::WIDTH)
+      density    = options.fetch(:density, Grim::DENSITY)
+      quality    = options.fetch(:quality, Grim::QUALITY)
       colorspace = options.fetch(:colorspace, Grim::COLORSPACE)
-      command = [@imagemagick_path, "-resize", width.to_s, "-antialias", "-render",
-        "-quality", quality.to_s, "-colorspace", colorspace,
-        "-interlace", "none", "-density", density.to_s,
-        "#{Shellwords.shellescape(pdf.path)}[#{index}]", path]
+      alpha      = options[:alpha]
+
+      command = []
+      command << @imagemagick_path
+      command << "-resize"
+      command << width.to_s
+      command << "-alpha #{alpha}" if alpha
```

Comment:
I just realized, following the pattern of the rest of the args here, that `-alpha` and then `#{alpha}` should be on separate lines like this:

```ruby
command << "-alpha" if alpha
command << "#{alpha}" if alpha
```